src/h5p-editor.js

```diff
@@ -21,10 +24,10 @@ class H5PEditor {
         keyValueStorage,
         config,
         libraryManager,
+        contentManager,
```

Comment:
Why do you pass the Managers and not just the Storages? Is there any reason why the Managers may be replaced with different objects?

Reply:
You're right you could simply construct new managers from the storages. The managers are required in the implementing module (h5p-demo) as well, though, as they grant access to the file streams, load library data and content data. The library manager will (probably?) be responsible for checking access rights of users so this functionality must be accessible in some way to the implementing module. I feel the only class that should call the storage objects should be the managers. That's why they are constructed in the implementation.

What do you think of calling the managers as properties of H5PEditor.Editor and passing in storage objects?

Reply:
I think that's a good idea since it would make the object instantiation simpler and would be consistent with how the ContentTypeCache and the ContentTypeInformationRepository are handled.

Reply:
I've done changes this now.